data-science-onramp/feature-engineering/feature_engineering.ipynb

```diff
@@ -1,4795 +1,1019 @@
 {
```

Comment:
<p>Using "clean" here might be confusing since we technically "cleaned" in the previous step of the tutorial. Maybe just a note at the beginning of the notebook explaining the difference?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahecole/python-docs-samples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I can use the word reformat or modify instead? Do you have a preference?</p>

Reply:
<p>Either works, I think. No preference!</p>

Reply:
<p>I used both!</p>